packages/core/src/integrations/eventFilters.ts

```diff
@@ -102,66 +107,72 @@ function _mergeOptions(
 }
 
 function _shouldDropEvent(event: Event, options: Partial<EventFiltersOptions>): boolean {
-  if (options.ignoreInternal && _isSentryError(event)) {
-    DEBUG_BUILD &&
-      logger.warn(`Event dropped due to being internal Sentry Error.\nEvent: ${getEventDescription(event)}`);
-    return true;
-  }
-  if (_isIgnoredError(event, options.ignoreErrors)) {
-    DEBUG_BUILD &&
-      logger.warn(
-        `Event dropped due to being matched by \`ignoreErrors\` option.\nEvent: ${getEventDescription(event)}`,
-      );
-    return true;
-  }
-  if (_isUselessError(event)) {
-    DEBUG_BUILD &&
-      logger.warn(
-        `Event dropped due to not having an error message, error type or stacktrace.\nEvent: ${getEventDescription(
-          event,
-        )}`,
-      );
-    return true;
-  }
-  if (_isIgnoredTransaction(event, options.ignoreTransactions)) {
-    DEBUG_BUILD &&
-      logger.warn(
-        `Event dropped due to being matched by \`ignoreTransactions\` option.\nEvent: ${getEventDescription(event)}`,
-      );
-    return true;
-  }
-  if (_isDeniedUrl(event, options.denyUrls)) {
-    DEBUG_BUILD &&
-      logger.warn(
-        `Event dropped due to being matched by \`denyUrls\` option.\nEvent: ${getEventDescription(
-          event,
-        )}.\nUrl: ${_getEventFilterUrl(event)}`,
-      );
-    return true;
-  }
-  if (!_isAllowedUrl(event, options.allowUrls)) {
-    DEBUG_BUILD &&
-      logger.warn(
-        `Event dropped due to not being matched by \`allowUrls\` option.\nEvent: ${getEventDescription(
-          event,
-        )}.\nUrl: ${_getEventFilterUrl(event)}`,
-      );
-    return true;
+  if (event.type === undefined) {
```

Comment:
```suggestion
  if (!event.type) {
```

saving some bytes? :D

Reply:
also, nit but maybe it would be nicer to invert this, so `if (event.type) { return false; }`, saving some nesting below